packit/api.py

```diff
@@ -1096,7 +1096,7 @@ def sync_release(
 
             commit_title, commit_description = get_commit_message_from_action(
                 output=commit_msg_action_output,
-                default_title=title or f"[packit] {version} upstream release",
+                default_title=title or f"rebase to {version} upstream release",
```

Comment:
in #2253 we are changing the default for changelog to `Update to version {version}`, I would probably stay consistent with that

Reply:
I just asked several maintainers about their opinion of Update vs. Rebase and the opinion was 3:2 for "update". Two people changed their opinion during discussion. And the debate was very heated :)
But the "update" is winner. I will amend this PR. :)